catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/taxonomy/CoreAttributes.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.impl.taxonomy;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.taxonomy.Core;
+import ddf.catalog.data.taxonomy.MetacardComponent;
+
+/**
+ * This class provides attributes that reflect core attributes expected to be relevant to all metacard types
+ * <p>
+ * https://codice.atlassian.net/wiki/display/DDF/Attributes
+ */
+public class CoreAttributes implements Core, MetacardComponent {
+    private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+
+    private static final String NAME = "core";
+
+    static {
+        DESCRIPTORS.add(new AttributeDescriptorImpl(CHECKSUM,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(CHECKSUM_ALGORITHM,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(CREATED,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.DATE_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(DESCRIPTION,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(EXPIRATION,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.DATE_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(ID,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
```

Comment:
That does not make sense for many of these and most of the current implementation cannot handle that.  You cannot have multiple metacard IDs for a single metacard.  There is only one metadata xml field that is parsed.  Multiple create dates does not make sense.  Most catalog providers today will not support that kind of change.  We really need to go back only set attributes that should be multivalued to true and the rest stay false.